components/region_cache_memory_engine/src/background.rs

```diff
@@ -121,6 +220,27 @@ impl GcRunner {
             "filtered_version" => filter.filtered,
         );
     }
+
+    fn gc_finished(&mut self) {
```

Comment:
Maybe it's a good idea to print some summary information on this round of GC--such as how many ranges, keys are GCed.  Print these lag ranges (whose snapshot is old but still running)---this is helpful to troubleshotting when a range may not gc at all. 

Meanwhile, I'm wondering for simplifying troubleshooting,  for reach range, we may add the tag information (since Key range is hard to troubleshoot), which contains region_id and its epoch when this range is initially loaded.

Reply:
https://github.com/tikv/tikv/blob/997eabc7f64d0bd1addc5d160e7d17cde1f52c3c/components/region_cache_memory_engine/src/gc.rs#L114  The summary information for each range is already have.